boltz/liquid.go

```diff
@@ -70,6 +70,14 @@ func (transaction *LiquidTransaction) FindVout(network *Network, addressToFind s
 
 }
 
+func (transaction *LiquidTransaction) VoutValue(vout uint32) (uint64, error) {
+	result, err := confidential.UnblindOutputWithKey(transaction.Outputs[vout], transaction.OurOutputBlindingKey.Serialize())
+	if err != nil {
+		return 0, err
+	}
+	return result.Value, nil
```

Comment:
This assumes that we always blind. Which is the case right now, but in the future it would be nice to not throw in case we don't
That adds some complexity in the construction of the transaction, so let's leave that for later